requirements.txt

```diff
@@ -1,4 +1,3 @@
-apache-airflow==1.10.2
 datadog==0.29.3
 google-api-python-client==1.7.9
```

Comment:
I think these dependencies might break composer. Maybe we should add them as a separate [google] tag.

Reply:
Okay, I removed all the dependencies other than datadog, should just be a clean install now and everything else is up to the user to install.